tests/test_web_urldispatcher.py

```diff
@@ -0,0 +1,104 @@
+import pytest
+import tempfile
+import aiohttp
+from aiohttp import web
+import os
+import shutil
+import asyncio
+
+# Timeout in seconds for an asynchronous test:
+ASYNC_TEST_TIMEOUT = 1
+
+
+class ExceptAsyncTestTimeout(Exception):
+    pass
+
+
+def run_timeout(cor, loop, timeout=ASYNC_TEST_TIMEOUT):
```

Comment:
Got it.
Right now aiohttp test suite has no such feature. I didn't feel the need for it.
Let's add the feature later to hooks that provide `@pytest.mark.run_loop` functionality.